### DISCUSSION TITLE
is it possible to setup Mqtt connection from a server to vs code extension ?

### DISCUSSION THREAD

**Discussion by iishika-s** [microsoft/vscode-discussions#757] [Extension Development QnA]
I am developing a Source control extension for vscode , and I want to establish an mqtt connection between extension and server, so that after activation of the extension if there is any event or change happened on server side, the extension will get the notification of it.

is this possible and how?

I am trying to look if, vs code allows to include such type of subscription to the server, without hampering the other processes in the extension.

I tried to execute a CLI command, which will keep listening to the server, but the executor method was just executing it and coming out of the method, it was not staying to read the sdt output, and if I tried to set an time interval, then other process were getting queued up

so now rather than executing any command, I want to create a method which will create mqtt connection with the server

**Comment by isidorn** [comment]
Yes, this should be possible. Did you try it? If yes, what issues did you hit?

**Reply by olguzzar** [reply]
Hi @iishika-s, any update on trying this?

**Comment by olguzzar** [comment]
Closing as there's been a period of inactivity. If there are any updates, feel free to reopen!
